pkg/accounting/tally/tally_test.go

```
@@ -61,11 +63,11 @@ func TestOnlineNodes(t *testing.T) {
 	assert.NoError(t, err)
 	defer func() { _ = accountingDb.Close() }()
 
-	bwDb, err := dbManager.NewDBManager("sqlite3", "file::memory:?mode=memory&cache=shared")
+	masterDB, err := satellitedb.NewDB("sqlite3://file::memory:?mode=memory&cache=shared")
 	assert.NoError(t, err)
-	defer func() { _ = accountingDb.Close() }()
+	defer func() { _ = masterDB.Close() }()
```

Don't ignore the errors.

---

I will reorganize tests in next pr